lock/src/main/java/com/auth0/android/lock/views/ClassicPanelHolder.java

```diff
@@ -357,11 +388,18 @@ public void onKeyboardStateChanged(boolean isOpen) {
         if (customFieldsForm != null) {
             customFieldsForm.onKeyboardStateChanged(isOpen);
         }
+        if (mfaCodeFormView != null) {
```

Comment:
Now seeing all these methods with the `if not nil then do something` for each form I think we need to handle this better since it doesn't scale and is highly error prone